sklearn/metrics/_classification.py

```diff
@@ -3278,9 +3278,9 @@ def d2_log_loss_score(y_true, y_pred, *, sample_weight=None, labels=None):
 
     Best possible score is 1.0 and it can be negative (because the model can be
     arbitrarily worse). A model that always uses the empirical mean of `y_true` as
-    constant prediction, disregarding the input features, gets a D^2 score of 0.0.
+    a constant prediction, disregarding the input features, gets a D^2 score of 0.0.
```

Comment:
Actually it should be
"A model that always predicts the per-class proportions, disregarding the input features, gets a D^2 score of 0.0."
We missed that during the review

Reply:
Done. Thanks